Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-alpine AS build
+# packages needed for node-gyp and dupe-krill
+RUN apk add --update --no-cache python make g++ rust cargo
+# use a non-root user to make commands such as npm post-install scripts safer
+USER node
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin:/home/node/.cargo/bin
+# RUN cargo install dupe-krill
+# copy by default will make the root user own the files, we want the node user to own the files
+COPY --chown=node:node package*.json .snyk /home/node/app/
+WORKDIR /home/node/app
+RUN npm install
+COPY --chown=node:node . /home/node/app/
+RUN npm run build
```

Comment:
The `build` task uses [`run-s`](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-s.md) to be on the path. I don't see where you've installed it.

Reply:
It also depends on the `production:eleventy` task, which requires [`eleventy`](https://github.com/11ty/eleventy) to be on the path.

Reply:
Lastly, `.eleventy.js` depends on `fast-glob`, which isn't listed in the dependencies. #2237 will resolve that.

Reply:
they are installed on like 12 `RUN npm install`

Reply:
`npm-run-all` (the thing that defines `run-s`) isn't in the project's `devDependencies`. Even if it were, `npm install` would install it to `./node_modules/` and its executable to `./node_modules/.bin/`, not to `/home/node/.npm-global/bin`. `npm install -g npm-run-all` would fix it. So would adding it to `devDependencies` and running it as `./nodes_modules/.bin/run-s …`

`@11ty/eleventy` is in `devDependencies`, but the rest of the above applies to its executable `eleventy` as well.

Reply:
the like 12 runs npm install which installs devdependencies and the build command is called via `npm run` which adds `./node_modules/.bin/ to the PATH, which is why this is working corectly

Reply:
I just tried again with your latest and no local changes. It works. I don't know what I was doing wrong. Resolved. 👍